tutorcredentials/plugin.py

```diff
@@ -22,7 +22,7 @@
         ("CREDENTIALS_VERSION", __version__),
         ("CREDENTIALS_BACKEND_SERVICE_EDX_OAUTH2_PROVIDER_URL", "http://lms:8000/oauth2"),
         ("CREDENTIALS_BACKEND_SERVICE_EDX_OAUTH2_KEY", "{{ CREDENTIALS_OAUTH2_KEY }}"),
-        ("CREDENTIALS_CATALOG_API_URL", "{{ LMS_HOST }}"),
+        ("CREDENTIALS_CATALOG_API_URL", "http://discovery.local.overhang.io/api/v1/"),
```

Comment:
This will work only locally, but not in production. On the other hand, it does not make sense to allow the user to modify this setting, so we should remove it from the config and hardcode the value in the init script. That value should probably be something like:

```
{% if ENABLE_HTTPS %}https{% else %}http{% endif %}{{ DISCOVERY_HOST }}/api/v1/
```

Reply:
good one